src/LeftNav/LeftNav.less

```diff
@@ -68,7 +74,7 @@
   // In order to have a background image (right arrow) and a background color
   // simultaneously, they both must be written in the same CSS rule and the
   // image must come second.
-  .tint(background, @primary_blue, 6);
+  .shade(background, @neutral_off_white, .15);
```

Comment:
Yeah, this was probably the first case where the existing steps didn't work at all. I've had a scale in my head, would be good to formalize eventually.